core/src/test/java/org/pentaho/di/connections/common/basic/TestBaseVFSConnectionDetails.java

```diff
@@ -0,0 +1,18 @@
+package org.pentaho.di.connections.common.basic;
+
+import org.pentaho.di.connections.vfs.BaseVFSConnectionDetails;
+import org.pentaho.di.core.variables.VariableSpace;
+
+public abstract class TestBaseVFSConnectionDetails extends BaseVFSConnectionDetails {
+
+  VariableSpace space;
+
+  @Override public VariableSpace getSpace() {
```

Comment:
Why have the annotations on the same line? Seems very different from most of the codebase (and often makes lines too long)

Reply:
addressed with https://github.com/pentaho/pentaho-kettle/commit/900fbd7f11c1f141a27963da6ddde07b0aa42206